react/features/conference/components/web/Subject.js

```diff
@@ -0,0 +1,67 @@
+/* @flow */
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { isToolboxVisible } from '../../../toolbox';
+
+/**
+ * The type of the React {@code Component} props of {@link Subject}.
+ */
+type Props = {
+
+    /**
+     * The subject of the conference.
+     */
+    _subject: string,
+
+    /**
+     * Indicates whether the component should be visible or not.
+     */
+    _visible: boolean
+};
+
+/**
+ * Subject react component.
+ *
+ * @class Subject
+ */
+class Subject extends Component<Props> {
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        const { _subject, _visible } = this.props;
+
+        return (
+            <div className = { `subject ${_visible ? 'visible' : ''}` }>
+                { _subject }
+            </div>
```

Comment:
Is anything visible if there is no subject?

Reply:
yes. the gradient. this is by design but maybe we would like to change it later.